opm/grid/cpgrid/processEclipseFormat.cpp

```diff
@@ -125,15 +126,20 @@ namespace cpgrid
         g.zcorn = &zcornData[0];
 
         g.actnum = actnumData.empty() ? nullptr : &actnumData[0];
+        std::map<int,int> nnc;
```

Comment:
The nncs are added to the grid itself in processEclipseFormat so we don't need them anymore. Maybe this will change in the future, if for instance we want to support output of ALLNNC, or we want to do support PINCH with the ALL option.